packages/input/SliderInput.tsx

```diff
@@ -12,19 +13,29 @@ import { SliderInputProps } from './types'
 function SliderInput({
   value,
   onChange,
+  onLabelClick,
   min = 0,
   max = 100,
   step = 1,
   minLabel,
   maxLabel,
   getLabel = (val) => String(val),
+  borderNone,
+  labels,
 }: SliderInputProps): React.ReactElement {
   const fillPercentage = ((value - min) / (max - min)) * 100
+  const LabelComponent = onLabelClick ? LabelButton : Label
+  const createClickHandler = (value: number) => {
```

Comment:
I would add empty line before and after 'createClickHandler' for better code readability